src/components/job-overview/style.css

```diff
@@ -0,0 +1,32 @@
+@value red from "../variables/colors.css";
+
+.note {
+  background-color: #fffcd7;
+  padding: 20px 20px 5px 20px;
+  box-shadow: 1px 1px 2px 2px rgba(0,0,0,0.2);
+  border: 1px solid #d0d0d0;
+  margin-bottom: 40px;
+  width: 30.5%;
```

Comment:
I think the width of this component should be determined by its surrounding component (like the grid cell component). Declaring this here makes it difficult to make this component responsive.